pkg/sql/opt/xform/testdata/rules/join

```diff
@@ -1454,6 +1454,35 @@ project
       │         └── const: 10 [type=int]
       └── filters (true)
 
+# Projection of constant columns work with non const expressions as well.
+exec-ddl
+CREATE TABLE bool_col (a INT, b INT, c bool, d bool, e bool, INDEX (a,b,c))
+----
+
+# Projection of constant columns work on boolean expressions.
+# Regression test for a test in opt-private-tests
```

Comment:
Please remove this line, we shouldn't have this kind of reference here. It's not important how we found the bug anyway.